docs/integrations/hypothesis.md

```diff
@@ -5,53 +5,6 @@ many standard library types, and generic types from the
 [`typing`](https://docs.python.org/3/library/typing.html) and
 [`typing_extensions`](https://pypi.org/project/typing-extensions/) modules by default.
 
-From Pydantic v1.8 and [Hypothesis v5.29.0](https://hypothesis.readthedocs.io/en/latest/changes.html#v5-29-0),
-Hypothesis will automatically load support for [custom types](/usage/types/) like
-`PaymentCardNumber` and `PositiveFloat`, so that the
-[`st.builds()`](https://hypothesis.readthedocs.io/en/latest/data.html#hypothesis.strategies.builds)
-and [`st.from_type()`](https://hypothesis.readthedocs.io/en/latest/data.html#hypothesis.strategies.from_type)
-strategies support them without any user configuration.
+Pydantic v2.0 drops built-in support for Hypothesis and no more ships with the integrated Hypothesis plugin.
```

Comment:
something about temporarily dropping it while we decide on the way forward, and a link to the issue I just created on annotated types.